src/JSInterop/Microsoft.JSInterop.JS/src/src/Microsoft.JSInterop.ts

```diff
@@ -264,17 +264,14 @@ module DotNet {
     }
 
     public serializeAsArg() {
-      return `__dotNetObject:${this._id}`;
+      return {__dotNetObject: this._id};
     }
   }
 
-  const dotNetObjectValueFormat = /^__dotNetObject\:(\d+)$/;
+  const dotNetObjectRefKey = '__dotNetObject';
   attachReviver(function reviveDotNetObject(key: any, value: any) {
-    if (typeof value === 'string') {
-      const match = value.match(dotNetObjectValueFormat);
-      if (match) {
-        return new DotNetObject(parseInt(match[1]));
-      }
+    if (value && typeof value === 'object' && value.hasOwnProperty(dotNetObjectRefKey)) {
```

Comment:
I need to publish this package to npm. How does one go about doing this @aspnet/build ?

Reply:
Send email with more details to our team so we can make sure get the setup done right. When does the package need to ship? Where? How does it get serviced?

Reply:
Opened https://github.com/aspnet/AspNetCore-Internal/issues/2541 to track the work to enable CD.